### DISCUSSION TITLE
Querying multiple object storage buckets from Thanos?

### DISCUSSION THREAD

**Discussion by ghost** [thanos-io/thanos#4823] [Questions & Answers]
Hi,

We are experimenting with Thanos and trying to understand how multiple object storage buckets can be queriable from one Thanos instance. We are using Kubernetes and the official Thanos helm chart as the base.

We have a number of Prometheus clusters with Thanos sidecars, all writing to different S3 buckets. What would be the easiest/recommended way to access the S3 data from these multiple buckets in one place at our central Thanos querier? We have no luck in adding more buckets in the objstoreconfig configuration and are struggling with finding more information or examples with multiple storegateway configurations. 

Thank you!

**Comment by GiedriusS** [comment]
The usual way to solve this issue is to start up multiple Thanos Store instances, each point to a different bucket. Hope that helps :+1:

**Reply by ghost** [reply]
Thank you! Is there any place I can find a good example of this setup with code?
